api/envoy/config/route/v3/route_components.proto

```diff
@@ -741,10 +741,14 @@ message RouteAction {
         "envoy.api.v2.route.RouteAction.UpgradeConfig";
 
     // Configuration for sending data upstream as a raw data payload. This is used for
-    // CONNECT requests, when forwarding CONNECT payload as raw TCP.
+    // CONNECT requests, when forwarding CONNECT payload as raw TCP. It also allows using POST
+    // payload as raw TCP if configured.
```

Comment:
This is not limited to `CONNECT` any more, would be better to re-word a bit.

Reply:
Reword, PTAL